src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/PolicyTests.ps1

```diff
@@ -78,3 +78,88 @@ function Test-PolicyAssignmentCRUD
 	Assert-AreEqual True $remove
 
 }
+
+<#
+.SYNOPSIS
+Tests Policy definition creation with parameters
+#>
+function Test-PolicyDefinitionWithParameters
+{
+	# Test
+	$actual = New-AzureRMPolicyDefinition -Name testPDWP -Policy "$TestOutputRoot\SamplePolicyDefinitionWithParameters.json" -Parameter "$TestOutputRoot\SamplePolicyDefinitionParameters.json"
+	$expected = Get-AzureRMPolicyDefinition -Name testPDWP
```

Comment:
I dont think it is a good idea to use another commandlet to give you the expected value.
If there is a bug that affects both the commandlets then it is likely it will go uncaught.